pulsar-functions/runtime/src/main/resources/java_instance_log4j2.yml

```diff
@@ -44,7 +44,7 @@ Configuration:
       name: RollingFile
       fileName: "${sys:pulsar.function.log.dir}/${sys:pulsar.function.log.file}.log"
       filePattern: "${sys:pulsar.function.log.dir}/${sys:pulsar.function.log.file}-%d{MM-dd-yyyy}-%i.log.gz"
-      immediateFlush: false
```

Comment:
I don't know if we want to change the immediateFlush to be true. May have performance impact. Anyone else have any thoughts?